doc/api/crypto.md

```diff
@@ -1041,6 +1041,26 @@ console.log(hash.digest('hex'));
 //   6a2da20943931e9834fc12cfe5bb47bbd9ae43489a30726962b576f4e3993e50
 ```
 
+### hash.copy([options])
```

Comment:
```suggestion
### hash.copy(\[options\])
```

(This will prevent `[options]` from being rendered as a link should there ever be a bottom-reference added for the text "options". This actually happened once with a different string and was only fixed in the last few days.)

Reply:
(Also, to answer the obvious question, a lint rule for that is in the works.)